frontend/packages/component-library/src/cms/snapshot/snapshot.module.scss

```
@@ -0,0 +1,48 @@
+@use '@code-dot-org/component-library-styles/colors.scss';
+@use '@code-dot-org/component-library-styles/mixins.scss' as mixins;
+@use '@code-dot-org/component-library-styles/typography.module.scss' as
+  typography;
+
+// SnapshotSection common styles
+.snapshot {
+  border-radius: 0.25rem;
+  border: 1px solid var(--borders-neutral-primary);
```

We have a border mixin you can use here:
```suggestion
  @include mixins.border-styles
```